### ISSUE TITLE
Broken since update 1.4

### ISSUE THREAD

**Issue by Celldweller** [enriquepiatti/Magicento#37]
I just updated to Magicento 1.4 and now it's totally broken for me. Can't use it anymore. :-/

21:39:17 PluginException: com/magicento/MagicentoProjectComponent : Unsupported major.minor version 51.0 [Plugin: Magicento]: com/magicento/MagicentoProjectComponent : Unsupported major.minor version 51.0 [Plugin: Magicento]

21:39:28 PluginException: update failed for AnAction with ID=GotoDeclaration: com/magicento/extensions/MagicentoGotoDeclarationHandler : Unsupported major.minor version 51.0 [Plugin: Magicento]

Using JetBrains 6.0 Build PS-127.100

System: MacOS 10.8.3

**Comment by jonpday** [comment]
it looks like v1.4 depends on the Factory functionality in PHPStorm 6.0.1 EAP - is there some way that 6.0.1 can be specified as a dependency for Magicento 1.4?

**Comment by martinhelfert** [comment]
Same for me.

PHPStorm 6.0.1 PS-129.196
MacOS 10.8.3
JDK 1.6.0_43

**Comment by maverick193** [comment]
Same for me 

09:36:53 PluginException: com/magicento/ui/forms/MagicentoSettingsForm : Unsupported major.minor version 51.0 [Plugin: Magicento]: com/magicento/ui/forms/MagicentoSettingsForm : Unsupported major.minor version 51.0 [Plugin: Magicento]

Using JetBrains PHPStorm 6.0
JDK 1.6.0_27

**Comment by steverice** [comment]
Same error, still on PHPStorm 5.0.4, JDK 1.6.0_43, OS X 10.8.3

```
com/magicento/MagicentoProjectComponent : Unsupported major.minor version 51.0 [Plugin: Magicento]: com/magicento/MagicentoProjectComponent : Unsupported major.minor version 51.0 [Plugin: Magicento]
```

**Comment by enriquepiatti** [comment]
Fixed in 1.4.2
